core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -27,11 +27,14 @@ class PolicyManager {
 
     private var cultureBuildingsAdded = HashMap<String, String>() // Maps cities to buildings
     private var specificBuildingsAdded = HashMap<String, MutableSet<String>>() // Maps buildings to cities
+
+    @Deprecated("Deprecated since 3.15")
     var autocracyCompletedTurns = 0
-    
+    var activeCombatBonuses = mutableSetOf<Pair<Float, Int>>()
```

Comment:
I think maybe change this to a countdown timer for all uniques, String to Int - maybe even in CivInfo
That way we can allow any variance in future countdown uniques
This may be way out of scope for this PR so probably just ignore this policy for now and we'll do it properly in another PR?

Reply:
If we change it, I'd rather do it in this PR, as then we don't have to support two different deprecated systems for the same feature. I'll think about the best approach some more

Reply:
I now implemented it so that each civilization has a list of 'temporary uniques'. These uniques are added together with an integer representing how many turns they are active. If this ends, it expires. Not sure if this is the best way to do it, but it works.